services/director-v2/src/simcore_service_director_v2/api/routes/dynamic_services.py

```diff
@@ -40,3 +51,45 @@ async def service_retrieve_data_on_ports(
     )
     # validate and return
     return RetrieveDataOutEnveloped.parse_obj(resp.json())
+
+
+@router.post(
+    "/{node_uuid}:start",
+    summary="start the dynamic-sidecar for this service",
```

Comment:
What kind of status errors this route can responde? Add ``responses``? Same applies for the others.